image-index.md

```diff
@@ -55,7 +55,10 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
     - **`os.version`** *string*
 
-        This OPTIONAL property specifies the operating system version, for example `10.0.10586`.
+        This OPTIONAL property specifies the version of the operating system included in the layers specified by the manifest.
```

Comment:
The operating system isn't included in the layers.  Maybe:

> This OPTIONAL property specifies the version of the operating system targeted by the referenced blob.

That also gets us away from using “manifest”, because while [`image-index.md` still talks about `manifests` as a list of manifests][1], the [non-manifest example in `image-layout.md`][2] shows that `manifests` is no longer restricted to just manifests.  I'd thought the wording around `manifests` was going to be generalized, but that doesn't seem to have happened yet.  I think changing the `manifests` wording is out of scope for this PR, but I'd rather not dig the current hole any deeper ;).

[1]: https://github.com/opencontainers/image-spec/blame/a82c09852aba4978e3d16bb0cd583d5e2f4a63c9/image-index.md#L25
[2]: https://github.com/opencontainers/image-spec/blame/a82c09852aba4978e3d16bb0cd583d5e2f4a63c9/image-layout.md#L187-L195